halo/evmstaking2/keeper/keeper_internal_test.go

```diff
@@ -165,10 +171,22 @@ func setupKeeper(
 	return k, ctx
 }
 
-type stakingKeeperMock struct{}
+type stakingKeeperMock struct {
+	// calls is the number of calls to GetValidator
+	calls uint32
```

Comment:
this will only ever be 0 or 1, and isn't technically correct since additional calls to GetValidator are not counted